lstchain/high_level/hdu_table.py

```diff
@@ -141,15 +135,14 @@ def create_hdu_index_hdu(
     base_dir = os.path.commonpath(
         [
             hdu_index_file.parent.absolute().resolve(),
-            fits_dir.absolute().resolve()
+            file_list[0].absolute().resolve()
```

Comment:
I'd say that `.absolute()` is not needed here. I think `.resolve()` already gives the absolute path.